src/stories/components/account/SigninSettings.stories.svelte

```diff
@@ -57,11 +115,15 @@
     title: "Sign in",
     customClass: "settings",
     headerClass: "settings",
+    isGoogle: false,
+    isWarning: false,
+    userEmail: "test@test.com",
   }}
 />
 
 <style>
   .sb-container {
     padding: 2rem 1rem;
+    display: block;
```

Comment:
oh the off-centering is because of this `display: block`. Is there another reason it's needed?

Reply:
ah i see; the width collapses if you remove it. yep I just investigated for a while and I have no idea what's going on there...